bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/gmos/GmosRule.java

```diff
@@ -291,7 +299,7 @@ public Problem check(Config config, int step, ObservationElements elems, Object
 //                        detectorManufacturer = SequenceRule.getInstrumentItem(config, InstGmosNorth.DETECTOR_MANUFACTURER_PROP);
 //                    }
 //                    // REL-231 - don't warn for GMOSN
-//                    return (amp != GmosCommonType.AmpCount.THREE) && (detectorManufacturer == GmosCommonType.DetectorManufacturer.E2V);
+//                    return (amp != AmpCount.THREE) && (detectorManufacturer == DetectorManufacturer.E2V);
```

Comment:
I'm guessing that was updated via find/replace.  The comment aside, it's nice to see the clutter of the `GmosCommonType` prefix removed.